docs/source/api_reference/transformations.rst

```diff
@@ -292,6 +292,14 @@ Depending on the transformer, the transformation parameters can be fitted.
 
     ScaledAsinhTransformer
 
+.. currentmodule:: sktime.transformations.series.dilation_mapping
+
+.. autosummary::
+    :toctree: auto_generated/
+    :template: class.rst
+
+    DilationMappingTransformer
```

Comment:
This is the wrong section, as the transform is not element-wise, it applies to entire series.

I can't spot a section to put it in, this is a resampling-like operation.
Do you have a good idea?

Perhaps we can change "Binning and segmentation" to "Binning, sampling, and segmentation"? Or, a new sampling section?

Reply:
Yes, you are right, I didn't really know where to put it. It is more like a permutation, but you can also view it as a particular type of sampling. I am thinking if there are other permutation-like things for time series, but the only thing I can think of is maybe for data augmentation, i.e., permuting close timesteps to add a little bit of noise. I don't know what's best for this case. Maybe use "Binning, sampling, and segmentation" for now and change it later if new approaches arise?

Reply:
sure. Augmentation though usually creates multiple series from one, this transformer creates one series from one, so it doesnt fully fit.